src/app/presenter/ChessBoardPresenter.ts

```diff
@@ -31,4 +33,13 @@ export class ChessBoardPresenter {
     const square: Square = { column: columns[position.x - 1], row: position.y as Row };
     return `${square.column.toLowerCase()}${square.row}`;
   }
+
+  private isPawnReadyForPromotion(position: Position): void {
+    const selectedSquare = this.translatePositionToAlgebraicNotation(position).toUpperCase();
+    const selectedSquareIsPawn = this.chessModel.squaresWithPiece[selectedSquare].name == 'Pawn';
+    if (selectedSquareIsPawn && (position.y == 1 || position.y == 8)) {
+      console.log('PromotionModalShouldBeOpened');
+      this.view.pawnPromotion();
```

Comment:
Dobrze, ze się zaczepiłaś o event, ale w międzyczasie zostało skończone zadanie:
https://github.com/nowakprojects/CodersCamp2020.Project.TypeScript.Chess/pull/80

Ale racja, brakuje jeszcze ruszania pionków od:
https://github.com/nowakprojects/CodersCamp2020.Project.TypeScript.Chess/pull/77

Bo my nie chcemy zareagowac na kilkniecie (jak teraz), tylko na ruszenie pionkiem :) 
Chociaz, z obecnym rozwiazaniem - tez to teraz bedzie dzialac. 

Mozna zawsze to zmergowac a potem przerobic.